### DISCUSSION TITLE
File temporaryUrl() returns 401 when passed as paramter

### DISCUSSION THREAD

**Discussion by DarpanKulkarni** [livewire/livewire#5447] [1. Help]
I have a blade component named 'image-preview.blade.php' with 'url' as a prop. When I pass '$file->temporaryUrl()' to this component it returns 401 error.

**Comment by joshhanley** [comment]
@DarpanKulkarni hmm that is odd!  Could you share your code which reproduces the issue with as little code as possible so we can see if we can assist? Thanks!
